vmaas/vulnerabilities.go

```diff
@@ -139,10 +139,20 @@ func evaluate(c *Cache, opts *options, request *Request) (*VulnerabilitiesCvesDe
 			}
 			seenPkgErratum[pe] = true
 			for _, cve := range c.ErratumDetails[update.Erratum].CVEs {
-				if _, inUnpatchedCves := cves.UnpatchedCves[cve]; inUnpatchedCves {
-					continue
+				if update.manuallyFixable {
+					if _, inUnpatchedCves := cves.UnpatchedCves[cve]; inUnpatchedCves {
+						continue
+					}
+					if _, inCves := cves.Cves[cve]; inCves {
```

Comment:
I think this can't be checked here because `cves.Cves` is built in this `for` cycle too. Manually fixable CVEs can be processed only after all fixable CVEs are processed.